src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: bold;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.searchForm {
+  font-family: inherit;
+  font-style: normal;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin: 20px auto;
+  line-height: 21.86px;
+}
+
+.input {
+  font-family: inherit;
```

Comment:
Please pay attention to CSS properties order
https://9elements.com/css-rule-order/

Reply:
I don`t fully understand the error. Can you explain it?

Reply:
Check the link I sent. Properties that are responsible for typography styles should be placed almost at the very bottom of the selector. Same with some other properties